rethinkdb/datadog_checks/rethinkdb/metrics/config.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+from typing import Iterator
+
+from ..connections import Connection
+from ..queries import QueryEngine
+from ..types import Metric
+
+logger = logging.getLogger(__name__)
```

Comment:
Does that work fine in the agent? I tought we were supposed to use the AgentCheck methods.

Reply:
The logs show fine, although the `check_id` is not present in the formatted log line:

```console
2020-03-13 18:05:27 UTC | CORE | DEBUG | (pkg/collector/python/check.go:72 in runCheck) | Running python check rethinkdb rethinkdb:5942258e6ff90a33
2020-03-13 18:05:27 UTC | CORE | DEBUG | (pkg/collector/python/datadog_agent.go:122 in LogMessage) | rethinkdb:5942258e6ff90a33 | (rethinkdb.py:90) | check config=Config(host='localhost', port=28015, user='datadog-agent', password='*****', tls_ca_cert=None)
2020-03-13 18:05:27 UTC | CORE | DEBUG | (pkg/collector/python/datadog_agent.go:122 in LogMessage) | rethinkdb:5942258e6ff90a33 | (rethinkdb.py:50) | connected connection_tags=['host:localhost', 'port:28015', 'server:server0', 'proxy:false']
```

See https://github.com/DataDog/integrations-core/pull/5715#discussion_r392390394

Reply:
I think the benefit of using `AgentCheck.log` is that you get the `check_id`, which can be useful for debugging:

https://github.com/DataDog/integrations-core/blob/3ab723d812fc79365ed4ada7e381d0f866813cc9/datadog_checks_base/datadog_checks/base/log.py#L34-L38

Reply:
We should pass around the class logger, id is useful for support cases

Reply:
Roger that, will address soon

Reply:
Addressed in 69686d23f0f321ef38b042207108c93336d97173